src/TestUtils/EventuallyConsistentTestTrait.php

```diff
@@ -17,7 +17,12 @@
 
 namespace Google\Cloud\TestUtils;
 
-use PHPUnit_Framework_ExpectationFailedException;
+if (!class_exists('\PHPUnit\Framework\ExpectationFailedException', true)) {
```

Comment:
Does this belong in `src/TestUtils/PHPUnit4To6Sim.php` too?

Reply:
This patch should work without installing the sim, so this is here.
This file should be usable for phpunit4 as well as phpunit6, without installing the sim.

Reply:
ok